nixos/tests/common/systemd-journal-remote/generate-certs.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -I nixpkgs=https://github.com/NixOS/nixpkgs/archive/nixpkgs-unstable.tar.gz -p minica -i bash
```

Comment:
If we still end up using this script somewhere, it should probably drop the shebang, and add a comment describing what packages it expects in $PATH.

If you run the script inside the VM, you can't download a random snapshot of nixpkgs from the internet.

Reply:
This script is only here to re-generate the certificates in the future. It's a file for nixpkgs developers, it should not be used in nix code.